src/org/omegat/filters3/xml/XMLTag.java

```diff
@@ -135,7 +136,7 @@ private boolean isDifferentDirection(boolean isRtl) {
             return false;
         }
 
-        return isRtl != Language.isRTL(sourceLanguage.getLanguageCode());
+        return isRtl != BiDiUtils.isRtl(sourceLanguage.getLanguageCode());
```

Comment:
I think there is no active reason to move the method `isRTL`, because there are other similar methods in `Language` class. It is better to keep there for backward compatibility.

Reply:
I checked a reason why we made Language class.
There is javadoc

```
 * This class is here, because the Locale has hard-coded '_' inside, and we must
 * adhere to ISO standard LL-CC.
 * <p>
 * This class tries to follow
 * <a href="https://www.gala-global.org/tmx-14b#xml:lang">TMX Specification on
 * languages</a>, which is based on
 * <a href="http://www.ietf.org/rfc/rfc3066.txt">RFC 3066</a>, i.e.
 * <ul>
 * <li>Language is composed from 1-8 alpha (A-Za-z) chars, then "-", then 1-8
 * alpha/digit chars (A-Za-z0-9).
 * <li>Case insensitive
 * <li>Case is not altered by this class, even though there exist conventions
 * for capitalization (ISO 3166 recommends that country codes are capitalized
 * (MN Mongolia), and ISO 639 recommends that language codes are written in
 * lower case (mn Mongolian)).
 * </ul>
```

It is for ISO standard LL-CC and RtL utility is no reason to place in Language class.